test/unit/specs/components/transactions/__snapshots__/TmLiStakeTransaction.spec.js.snap

```diff
@@ -308,17 +308,17 @@ exports[`TmLiStakeTransaction unbonding delegations should show unbondings and c
         >
           <div>
             
-      Unbonded 
+      Undelegated 
             <b>
               1000
             </b>
             <span>
                stakes
             </span>
             <span
-              class="tx-unbonding__time-dif"
+              class="tx-unbonding__time-diff"
             >
-               - a few seconds ago
+               (liquid a few seconds ago)
```

Comment:
I don't understand the phrase: `liquid a few seconds ago`

Reply:
woah, nice catch. 

lol yes, because it doesn't make any sense 😆 

i will fix this. i assumed `.timeDiff()` would only be used for dates in the future.

Reply:
i was not able to successfully fix this snapshot. tried many things - but moment insisted on returning the same message. could use a hand.

Reply:
well, i added the word "liquid" - so that's correct ... but no matter what `unbondingTime` i set with `setProps` `moment.fromNow()` returns `in a few seconds` ... in the app (outside of the tests) `timeDiff` will return something like `liquid in 3 days` ... which is clear and true IMO ... but i can't get moment to work _in the tests_.